apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/ui/customizer/CustomizerSources.java

```diff
@@ -204,4 +209,23 @@ private void initAccessibility() {
         prjFolderValue.getAccessibleContext().setAccessibleDescription(getMessage("ACS_PrjFolderValue"));
     }
     
+    private String[] sourceLevels(JavaPlatform platform) {
+        List<String> levels = new ArrayList<>();
+        levels.add("1.4");
+        levels.add("1.5");
+        levels.add("1.6");
+        levels.add("1.7");
+        levels.add("1.8");
+        try {
+            String platformVersion = platform.getSpecification().getVersion().toString();
+            int maxLevel = Integer.parseInt(platformVersion.split("\\.")[0]);
+            for (int level = 9; level <= maxLevel; level++) {
+                levels.add(Integer.toString(level));
+            }
+        } catch (Exception ex) {
+            Exceptions.printStackTrace(ex);
+        }
+        return levels.toArray(String[]::new);
```

Comment:
that would be Java 11 API. needs to be `new String[0]`

Reply:
Good catch!  Getting mixed up with `Stream`. Doesn't help that `-release` isn't used because `requires.nb.javac=true` customizes the boot classpath.